ariadne/src/Ariadne/Wallet/Cardano/Kernel/PrefilterTx.hs

```diff
@@ -195,12 +205,23 @@ prefilter (wid,hdPass) selectAddr rtxs
   Pretty-printing
 -------------------------------------------------------------------------------}
 
-instance Buildable PrefilteredBlock where
-  build PrefilteredBlock{..} = bprint
-    ( "PrefilteredBlock "
-    % "{ inputs:  " % listJson
-    % ", outputs: " % mapJson
-    % "}"
-    )
-    (Set.toList pfbInputs)
-    pfbOutputs
+-- TODO: how to avoid introducing orphans here?
```

Comment:
It's not necessary to define `instance Buildable AddrWithId` in order to define `Buildable Prefiltered{Block,Utxo}`.